UM/Qt/QtApplication.py

```diff
@@ -263,6 +263,9 @@ def _onJobFinished(self, job: Job) -> None:
         elif (not isinstance(job, ReadMeshJob) and not isinstance(job, ReadFileJob)) or not job.getResult():
             return
 
+        if isinstance(job, WriteFileJob) and not job.getFileName().endswith(".curaproject.3mf"):
```

Comment:
Uranium doesn't know about Cura, so we should not put this here. And we just want to above gcode here.